ocaml/xapi/xapi.ml

```diff
@@ -1138,7 +1153,57 @@ let server_init () =
               (* Try to say hello to the pool *)
               match attempt_pool_hello ip with
               | None ->
-                  finished := true
+                  let obj_uuid = Helpers.get_localhost_uuid () in
+                  let message_name, _ =
+                    Api_messages
+                    .xapi_startup_blocked_as_version_higher_than_coordinator
+                  in
+                  let existing_alerts =
+                    Helpers.call_api_functions ~__context (fun rpc session_id ->
+                        Client.Client.Message.get_all_records ~rpc ~session_id
```

Comment:
This call may fail. The failure has to be handled properly.
Basically, this failure should not block the checking.